src/FixedDataTableCell.js

```diff
@@ -108,8 +108,8 @@ class FixedDataTableCell extends React.Component {
       return true;
     }
 
-    const { cell: oldCell, isScrolling: oldIsScrolling, ...oldProps } = this.props;
-    const { cell: newCell, isScrolling: newIsScrolling, ...newProps } = nextProps;
```

Comment:
Removing this so that when we stop scrolling we will re-render.  What could happen before is other prop changes while scrolling would be dropped and we weren't re-rendering once the scroll stopped.